scala/BUILD

```diff
@@ -94,3 +96,36 @@ java_library(
     srcs = ["PlaceHolderClassToCreateEmptyJarForScalaImport.java"],
     visibility = ["//visibility:public"],
 )
+
+toolchain_type(
+    name = "deps_toolchain_type",
+    visibility = ["//visibility:public"],
+)
+
+toolchain(
+    name = "deps_toolchain",
+    toolchain = "@io_bazel_rules_scala//scala:deps_toolchain_impl",
+    toolchain_type = "@io_bazel_rules_scala//scala:deps_toolchain_type",
+    visibility = ["//visibility:public"],
+)
+
+declare_deps_provider(
+    name = "scala_xml_provider",
+    visibility = ["//visibility:public"],
+    deps = ["//external:io_bazel_rules_scala/dependency/scala/scala_xml"],
```

Comment:
is this for backwards compatibility? if so have you thought of how/when we should break it?

Reply:
yes, it's for backwards compatibility. To get rid of it we need to provide users with an alternative for binds (`*_repositories` without binds, toolchains, or something loader specific).